source/Calamari.Tests/AWS/CloudFormation/CloudFormationFixtureHelpers.cs

```diff
@@ -181,8 +185,8 @@ public void DeployTemplateS3(string resourceName, IVariables variables)
                                                               }, variables, fileSystem, log));
                 var result = command.Execute(new[]
                 {
-                    "--templateS3", "https://octopus-cloudformation-s3-test.s3.amazonaws.com/empty.yaml",
-                    "--templateS3Parameters", "https://octopus-cloudformation-s3-test.s3.amazonaws.com/properties.json",
+                    "--templateS3", "https://calamari-cloudformation-tests.s3.amazonaws.com/s3/empty.yaml",
+                    "--templateS3Parameters", "https://calamari-cloudformation-tests.s3.amazonaws.com/s3/properties.json",
```

Comment:
The old `octopus-cloudformation-s3-test` bucket lived in the main Octopus Deploy AWS account, which this no longer has permissions to.

Created a new bucket in the E2E test account and copied the files in there (slightly different structure to avoid name clashes)

Reply:
@APErebus  so this bucket won't be deleted as part of the runbook, correct?

Reply:
Correct. The runbook has a specific naming scheme it's looking for